doc/usage/restructuredtext/domains.rst

```diff
@@ -231,6 +231,14 @@ The following directives are provided for module and class contents:
       Describe the location where the object is defined.  The default value is
       the module specified by :rst:dir:`py:currentmodule`.
 
+
```

Comment:
```suggestion
   .. _pyfuncsinglelinesig:

```

Reply:
Found it! Can be linked with 
```
:rst:dir:`py:function:single-line-signature`
```
The text of the link is then **py:function:single-line-signature**

Reply:
🥇